### DISCUSSION TITLE
using anchor-stiker component in base theme

### DISCUSSION THREAD

**Discussion by RomanMG** [vuepress-theme-hope/vuepress-theme-hope#605] [Q&A] [answered]
Hi everyone!

Unfortunately I'am new in vue.js and in vuepress and maybe my question wil be funny and silly...

Can you help and explain for dummie how possible to use anchor-stiker on the right sidebar from this theme [https://vuepress-theme-hope.github.io/guide/interface/icon/](url) 
![anchorSticker](https://user-images.githubusercontent.com/23496551/117552337-5e8cec00-b053-11eb-8d3d-dfba6629bfee.png)

in the standart theme vuepress? Maybe someone have right-anchor plugin to install...
Many thanks for help!

P.S.
I found some plugins such 'vuepress-plugin-right-anchor' but it has two things which is now suitable. It's not closed where the screen become small and it's not depending from the width of the content.
looks like:
![mobile_bad](https://user-images.githubusercontent.com/23496551/117552448-05718800-b054-11eb-91f6-004c5211b6e3.png)

**Comment by Mister-Hope** [comment, marked as answer]
This is a repo for `vuepress-theme-hope`.

Maybe you should go to official repo or the plugin repo for help.

Solution:

You can add a `@media` selector on the right anchor to hide it on mobile. For details please refer to MDN.

BTW, the theme is providing this features, and  could you give some feedbacks why you decide not to use my theme? Thanks.

**Reply by RomanMG** [reply]
I'am ready to use vuepress-theme-hope:) But I can't understand what I'am doing wrong.
I' am install theme using: npm i vuepress-theme-hope . I see files in folder "node_modules\vuepress-theme-hope", put this code: const { config } = require("vuepress-theme-hope"); in config.js but nothing happend, I see standart theme:(

Thanks for your help!

**Reply by Mister-Hope** [reply]
Show me your code or your repo link if you want more help from me. I can only help with enough infomation.

**Reply by Mister-Hope** [reply]
One more thing, the theme is only full tested on webkit browsers (Safari, New Microsoft Edge and Chrome). It should work on firefox (the sreenshot seems coming from firefox). But the appearance in your screenshot is definitely having problems.

Please use latest Chrome to open devServer to prevent compatibility problems

**Reply by RomanMG** [reply]
Thank you!

I'm find the problem, it was in config.js, right code below:
// .vuepress/config.js
const { config } = require("vuepress-theme-hope");

module.exports = config({
  plugins: ['fulltext-search'],
  locales: {
    "/": {
      // set language you want
      lang: "en-US",
    },
  },
});

now it look like:
![all_ok](https://user-images.githubusercontent.com/23496551/117572376-4660af80-b0db-11eb-9cfb-ea85e3be51f9.png)

Thank you for your attention to my question!
Have a good day!
